sandbox/apparmor/apparmor_test.go

```diff
@@ -283,6 +308,24 @@ profile snap-test {
 	features, err := apparmor.ProbeParserFeatures()
 	c.Check(err, Equals, os.ErrNotExist)
 	c.Check(features, DeepEquals, []string{})
+
+	// pretend we have an internal apparmor_parser
+	fakeroot := c.MkDir()
+	dirs.SetRootDir(fakeroot)
```

Comment:
for cleanliness this needs to be undone with:

defer dirs.SetRootDir("")

seems there are other and older calls that needs this treatment

Reply:
Thanks - added in https://github.com/snapcore/snapd/pull/12087/commits/cef4cfecbd0d2f673151ba9cb87a258d85e07d83